src/Front/Parser.y

```diff
@@ -127,8 +138,8 @@ patAtom                               --> Pat
   : '_'                                 { PatWildcard }
   | '(' patTups ')'                     { normalize id PatTup $2 }
   | '(' ')'                             { PatLit LitEvent }
+  | int                                 { PatLit $ LitInt $1 }
```

Comment:
We now support integer patterns, as well as data constructor patterns via `id`.

Something like `f ((X y) z) = ...` is syntactically valid but we will catch it later in `Scope.hs`

Reply:
Does sslang support booleans all the way through? Is it worth throwing in bool literals while you're already here?